TLM/TLM/State/SavedGameOptions.cs

```diff
@@ -112,10 +112,19 @@ public class SavedGameOptions {
     public static bool Available { get; set; } = false;
 
     public static SavedGameOptions Instance { get; private set; }
+
+    /// <summary>
+    /// ensures SavedGameOptions exists.
+    /// if not then it is set to global default or built-in default (if global default does not exist).
+    /// </summary>
     public static void Ensure() {
         Log.Info("SavedGameOptions.Ensure() called");
         if (Instance == null) {
-            Create();
+            if (GlobalConfig.Instance?.SavedGameOptions != null) {
```

Comment:
How that applies only to "new game"? It applies to everything, doesn't matter which mode user selected - start new game, load, enter map editor, scenario, and even if you restart game since you've put it into global config as a public instance field ((de)serialized by default).

Reply:
>  start new game, ~~load~~, enter map editor, scenario

No. when user loads saved game then built-in/default options are overwritten with saved values.
one option would be to  put a condition: if loading saved game then set to built in default until OptionsManager deserializes the values. If loading new game then set to default for new game.

But there is no point for such a check. it does not matter what I default it to, the values are overwritten.

Reply:
Good 👍